frontend/packages/dev-console/src/extensions/add-actions.ts

```diff
@@ -0,0 +1,29 @@
+import { Extension } from '@console/plugin-sdk/src/typings/base';
+import { AccessReviewResourceAttributes } from '@console/internal/module/k8s';
+
+namespace ExtensionProperties {
+  export type AddAction = {
+    /** Optional ID used to identify the action. */
+    id?: string;
+    /** The label of the actio.n */
+    label: string;
+    /** The description of the action.  */
+    description: string;
+    /** The perspective display icon. */
+    icon?: string;
+    /** The perspective display icon css class. */
+    iconClass?: string;
+    /** The URL to navigate to. */
+    url: string;
+    /** Optional access review to control visibility / enablement of the action. */
+    accessReview?: AccessReviewResourceAttributes | AccessReviewResourceAttributes[];
```

Comment:
Based on my suggestion above:

```suggestion
    accessReview?: AccessReviewResourceAttributes[];
```

Type simplicity reflects into simplicity of code that interprets the values.

Reply:
I'll simplify the type to just an array.